vault/resource_ad_secret_backend.go

```diff
@@ -6,16 +6,14 @@ package vault
 import (
 	"context"
 	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"log"
 	"strings"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-provider-vault/internal/consts"
 	"github.com/hashicorp/terraform-provider-vault/internal/provider"
 	"github.com/hashicorp/terraform-provider-vault/util"
-	"github.com/hashicorp/terraform-provider-vault/util/mountutil"
-
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/hashicorp/vault/api"
 )
 
 func adSecretBackendResource() *schema.Resource {
```

Comment:
Since the AD resources are deprecated, I wonder if we should not update this? There is no harm in it, per se. But I don't think we want to give the wrong idea.

Reply:
Have they officially been deprecated? I thought we had previously decided to delay the deprecation because users were having trouble migrating over to OpenLDAP, but the decision may have changed since then, so please let me know if I'm not up to date 🙏🏼

Reply:
End of support for AD secrets is this quarter and remove is the begging of next year.

https://developer.hashicorp.com/vault/docs/deprecation#active-directory-secrets-engine